kDrive/AppDelegate.swift

```diff
@@ -221,6 +221,19 @@ final class AppDelegate: UIResponder, UIApplicationDelegate, AccountManagerDeleg
                 StorePresenter.showStore(from: viewController, driveFileManager: driveFileManager)
             }
             return true
+        } else if components.host == "file",
```

Comment:
Nothing really, but I would prefer to use a switch on `components.host`. It's a subjective thing. As you prefer.

Reply:
discussed, not helpful here.